web3.md

```diff
@@ -503,7 +503,7 @@ WEB3 JSON RPC
     syntax KItem ::= "#eth_sendTransaction"
                    | "#eth_sendTransaction_finalize"
  // ------------------------------------------------
-    rule <k> #eth_sendTransaction => loadTX J ~> #eth_sendTransaction_finalize ... </k>
+    rule <k> #eth_sendTransaction => createTX !ID:Int J ~> #eth_sendTransaction_finalize ... </k>
```

Comment:
Is the fresh `ID` guaranteed to be unused by another transaction? I assume it would need some more context here.

Reply:
`!ID:Int` will be a fresh integer, so as long as we don't put transactions in the queue except by using `createTX` this should be OK.

Reply:
The fresh `ID` is now generated inside `createTX`.